ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/table-export/table-export.tsx

```diff
@@ -82,6 +102,7 @@ export default hot(module)((props: Props) => {
         buttonType={buttonTypeEnum.primary}
         icon="fa fa-download"
         text="Download"
+        disabled={customExportCount > properties.exportResultLimit}
```

Comment:
If a user enters a number greater than the limit then switches the export type to "All" or "Visible", is the button still disabled?

Reply:
Oh, yeah.

Good call on this one.

Fixed to only disable on the custom option.